apply/run.go

```diff
@@ -109,8 +109,12 @@ func getHosts(inMasters, inNodes string) ([]v2.Host, error) {
 	}
 
 	masters := strings.Split(inMasters, ",")
-	masterHosts := make([]v2.Host, len(masters))
+	var masterHosts []v2.Host
 	for index, master := range masters {
+		if master == "" {
```

Comment:
This part should be validated in the validation layer. In the processing layer, no master and node would be empty string. @VinceCui